src/xcode/ENA/ENA/Source/Extensions/PDFDocument+EmbedImage.swift

```diff
@@ -13,7 +13,7 @@ extension PDFDocument {
 	
 	/// Embeds an Image on the first Page on given position
 	/// Inspired  by https://pspdfkit.com/blog/2019/insert-image-into-pdf-with-swift/
-	func embed(image: UIImage, at position: CGPoint) throws {
+	func embed(image: UIImage, at position: CGPoint, text: String, of size: CGFloat?, hex color: String, with textRect: CGRect) throws {
```

Comment:
Since we embed more then an image it would be nice to rename the extension to something more fitting

Reply:
Also we should use native UIColor instead of hex strings. Do the conversion from hex string to UIColor as early as possible

```suggestion
	func embed(image: UIImage, at position: CGPoint, text: String, of size: CGFloat?, color: UIColor, with textRect: CGRect) throws {
```

Reply:
Will fix this :)